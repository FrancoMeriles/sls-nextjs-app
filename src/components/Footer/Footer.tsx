import { Flex, Text } from '@chakra-ui/react'

const Footer = () => {
  return (
    <Flex pt={10} pb={10} align="center" justify="center">
      <Text color="gray.300" fontSize="xs">
        Copyright Â© 1999-2021 MercadoLibre S.R.L.
      </Text>
    </Flex>
  )
}

export default Footer
