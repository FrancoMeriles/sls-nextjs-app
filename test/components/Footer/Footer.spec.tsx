import { render, screen } from '@testing-library/react'
import renderer from 'react-test-renderer'

import Footer from '@base/components/Footer'

describe('Footer', () => {
  it('should render and match snapshot with required props', () => {
    const tree = renderer.create(<Footer />).toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('should renders and display the correct data', async () => {
    render(<Footer />)
    expect(screen.getByText('Copyright Â© 1999-2021 MercadoLibre S.R.L.')).toBeInTheDocument()
  })
})
