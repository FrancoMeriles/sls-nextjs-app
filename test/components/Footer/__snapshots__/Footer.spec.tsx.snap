// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render and match snapshot with required props 1`] = `
<div
  className="css-3c83ig"
>
  <p
    className="chakra-text css-19z3xop"
  >
    Copyright Â© 1999-2021 MercadoLibre S.R.L.
  </p>
</div>
`;
